jenkins/scripts/skip_integration_test.sh

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env bash
+
+set -eu
+
+exclude_md() {
+  for file in $(git diff "${ghprbActualCommit}" origin/"${ghprbTargetBranch}" --name-only)
```

Comment:
Which git repo is used here? Is it project-infra?

Reply:
The repo where integration test will be triggered

Reply:
But I think we always start with project-infra and then the other repos are cloned in `integration_test.sh`. Why I'm asking is because I don't see any `cd` or similar so I think we will still be in the repo where the `*.pipeline` file is.

Reply:
Good point, I missed that part, thanks. Now I changed it, so we clone the $UPDATED_REPO where the PR was open, cd to repo and do git diff.